src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/help-wait.groovy

```diff
@@ -0,0 +1,10 @@
+p('''
+    You may ask that this Pipeline build wait for completion of the downstream build.
+    In that case the return value of the step is an object on which you can obtain the following read-only properties:
+    so you can inspect its <code>result</code> and so on.
+''')
+raw(org.jenkinsci.plugins.workflow.support.steps.build.RunWrapper.class.getResource("RunWrapper/help.html").text)
```

Comment:
Tried using `st:include` but without success, since `RunWrapper` is not a `Descriptor` so it has no URL-binding out of the box; anyway looking up `RunWrapper.class` or `RunWrapper.DescriptorImpl.class` from Jelly is too hard.